### ISSUE TITLE
docs: directive(s) not present in typeDefs when using buildTypeDefsAndResolvers(Sync)

### ISSUE THREAD

**Issue by carlocorradini** [MichalLytek/type-graphql#1399]
I think we should add to the docs that `buildTypeDefsAndResolvers` and `buildTypeDefsAndResolversSync` do not have any **directive**(s) in `typeDefs`.

**Comment by MichalLytek** [comment]
We could even replace the printSchema with the function from graphql tools to have directives printed.

**Comment by carlocorradini** [comment]
Therefore, add graph-tools as a dependency?
This could solve all directive(s) issues :)

**Comment by MichalLytek** [comment]
Yes, until we have a plugin system and modular packages, we need to download everything even if not needed for most of the users 🤔

**Comment by carlocorradini** [comment]
Any idea for the implementation of a plugin/extension system?

**Comment by MichalLytek** [comment]
Started 2 years ago a complete rewrite from scratch 😅

**Comment by carlocorradini** [comment]
Yup, saw it!
All the ideas are implemented in v2? Or you simply added them as idea in GitHub project?

**Comment by MichalLytek** [comment]
The board was up to date with the state of my local branch. Now I think I would need to update lerna to nx, npm workspaces, etc. 😄

**Comment by carlocorradini** [comment]
If I can help, I love TypeGraphQL and I'm using it in my Master's Thesis (and a lot of other projects too), so... yeah, TypeGraphQL all the way 😂

Several months ago I may have agreed with you  for ditching Lerna with something else but now Lerna is super carried and heavily maintained.
Do you want to use Lerna for `packages` (i.e., plugins, TypeGraphQL ecosystem), I've also used `npm workspaces`, and please let me know if I can assist...

**Comment by carlocorradini** [comment]
Moreover, I think that could be super useful if we create a TypeGraphQL chatroom (i.e., Discord, Slack, etc.) for thoughts, etc.
